index.d.ts

```diff
@@ -1067,10 +1067,10 @@ export interface LanguageDetectorAsyncModule extends Module {
   type: 'languageDetector';
   /** Set to true to enable async detection */
   async: true;
-  init(services: Services, detectorOptions: object, i18nextOptions: InitOptions): void;
+  init(services: Services, detectorOptions: object, i18nextOptions: InitOptions): void | Promise<void>;
```

Comment:
the init option does not return a promise

Reply:
In my case, `init` no need return `promise`. But I think in some case, it will need.
Because this is module with `async: true`.
That is reason its name is `LanguageDetectorAsyncModule`, not `LanguageDetectorModule`
Please check my below comment